admin/admin_edit.php

```diff
@@ -84,6 +84,7 @@
             } else unset($_REQUEST['password']);
 
             if (!$admin_id) $admin_id=time();
+            foreach (array('fname','lname') as $k) $_REQUEST[$k]=trim($_REQUEST[$k]);
```

Comment:
Ben, three things here:

1. to what extent we follow the https://github.com/orsee/orsee/wiki/Coding-style, if it is strict then this line should have brackets to make it easier to follow.
2. I am not sure if array values trimming needs to be done here in one-by-one fashion, it should be applied to any array passed as the first parameter within orsee_db_save_array function, if that is possible (unless there is a case when text value needs to contain spaces at either end).
3. "Username ist lost" <- typo in commit message.